pkg/ccl/changefeedccl/sink_cloudstorage.go

```diff
@@ -36,8 +36,9 @@ import (
 
 func isCloudStorageSink(u *url.URL) bool {
 	switch u.Scheme {
-	case `experimental-s3`, `experimental-gs`, `experimental-nodelocal`, `experimental-http`,
-		`experimental-https`, `experimental-azure`:
+	case changefeedbase.SinkSchemeCloudStorageS3, changefeedbase.SinkSchemeCloudStorageGCS,
+		changefeedbase.SinkSchemeCloudStorageNodelocal, changefeedbase.SinkSchemeCloudStorageHTTP,
+		changefeedbase.SinkSchemeCloudStorageHTTPS, changefeedbase.SinkSchemeCloudStorageAzure:
```

Comment:
Set these as constants for now. Will be easier to track when we change the `experimental-` prefix